packages/message/src/Executions.ts

```diff
@@ -147,3 +158,81 @@ export const getMessageExecutionParentArn = (
 export const getMessageCumulusVersion = (
   message: Message.CumulusMessage
 ): string | undefined => message.cumulus_meta?.cumulus_version;
+
+/**
+ * Get the workflow tasks a workflow message, if any.
```

Comment:
This is a nit but that comment confuses me. Did you mean `Get the workflow tasks'?

Reply:
I meant to say "Get the workflow tasks in a workflow message". Would that be better?

Reply:
Yes, that works

Reply:
038e5edd614f837d0ef0c20e25490ce998457d03